index.bs

```diff
@@ -273,6 +273,7 @@ The <dfn method for="XR">supportsSession(|mode|)</dfn> method queries if a given
 When this method is invoked, it MUST run the following steps:
 
   1. Let |promise| be [=a new Promise=].
+  1. If the requesting document's origin is not allowed to use the WebXR [[#feature-policy|feature policy]], [=reject=] |promise| with a "{{SecurityError}}" {{DOMException}} and abort these steps.
```

Comment:
It looks like this is the supportsSession algorithm, should this be done for requestSession as well?

Reply:
Ahh, requestSession I see that it's buried in the "Trustworthy" and "request inline" sessions.  Perhaps it should be unified up here to match the early bail out of supportsSession?
Also, if the decision is made to *not* make this change for whatever reason, should that logic be removed?